scan/lib/scan/options.rb

```diff
@@ -97,6 +97,12 @@ def self.available_options
                                      env_name: "SCAN_BUILDLOG_PATH",
                                      description: "The directory were to store the raw log",
                                      default_value: "#{FastlaneCore::Helper.buildlog_path}/scan"),
+        FastlaneCore::ConfigItem.new(key: :include_simulator_logs,
+                                     env_name: "SCAN_INCLUDE_SIMULATOR_LOGS",
+                                     description: "Should the logs of the app on the Simulator be written to the output_directory",
+                                     type: TrueClass,
```

Comment:
Do you mean does this ConfigItem actually work? It should as I used the same code for [import_certificate](https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/import_certificate.rb#L35), but I have not created any tests against it and would like to know where the best place would be to write such a test (I don't see specific tests for options in `scan`)